ARM.CMSIS.pdsc

```diff
@@ -1114,897 +1125,18 @@ and 8-bit Java bytecodes in Jazelle state.
-
-    <!-- ARMCC compiler -->
-    <condition id="CA_ARMCC5">
-      <description>Cortex-A5, Cortex-A7 or Cortex-A9 processor based device for the Arm Compiler 5</description>
-      <require condition="ARMv7-A Device"/>
-      <require condition="ARMCC5"/>
-    </condition>
-    <condition id="CA_ARMCC6">
-      <description>Cortex-A5, Cortex-A7 or Cortex-A9 processor based device for the Arm Compiler 6</description>
-      <require condition="ARMv7-A Device"/>
-      <require condition="ARMCC6"/>
-    </condition>
-
-    <condition id="CM0_ARMCC">
-      <description>Cortex-M0 or Cortex-M0+ or SC000 processor based device for the Arm Compiler</description>
-      <require condition="CM0"/>
-      <require Tcompiler="ARMCC"/>
-    </condition>
-    <condition id="CM0_ARMCC5">
-      <description>Cortex-M0 or Cortex-M0+ or SC000 processor based device for the Arm Compiler 5</description>
-      <require condition="CM0"/>
-      <require condition="ARMCC5"/>
-    </condition>
-    <condition id="CM0_ARMCC6">
-      <description>Cortex-M0 or Cortex-M0+ or SC000 processor based device for the Arm Compiler 6</description>
-      <require condition="CM0"/>
-      <require condition="ARMCC6"/>
-    </condition>
-    <condition id="CM0_LE_ARMCC">
-      <description>Cortex-M0 or Cortex-M0+ or SC000 processor based device in little endian mode for the Arm Compiler</description>
-      <require condition="CM0_ARMCC"/>
-      <require Dendian="Little-endian"/>
-    </condition>
-    <condition id="CM0_BE_ARMCC">
-      <description>Cortex-M0 or Cortex-M0+ or SC000 processor based device in big endian mode for the Arm Compiler</description>
-      <require condition="CM0_ARMCC"/>
-      <require Dendian="Big-endian"/>
-    </condition>
-
-    <condition id="CM1_ARMCC">
-      <description>Cortex-M1 based device for the Arm Compiler</description>
-      <require condition="CM1"/>
-      <require Tcompiler="ARMCC"/>
-    </condition>
-    <condition id="CM1_ARMCC5">
-      <description>Cortex-M1 based device for the Arm Compiler 5</description>
-      <require condition="CM1"/>
-      <require condition="ARMCC5"/>
-    </condition>
-    <condition id="CM1_ARMCC6">
-      <description>Cortex-M1 based device for the Arm Compiler 6</description>
-      <require condition="CM1"/>
-      <require condition="ARMCC6"/>
-    </condition>
-    <condition id="CM1_LE_ARMCC">
-      <description>Cortex-M1 based device in little endian mode for the Arm Compiler</description>
-      <require condition="CM1_ARMCC"/>
-      <require Dendian="Little-endian"/>
+    <!-- Startup -->
+    <condition id="Startup ARMCC6 Secure">
+        <require condition="ARMCC6"/>
+        <require condition="TZ Secure"/>
     </condition>
-    <condition id="CM1_BE_ARMCC">
-      <description>Cortex-M1 based device in big endian mode for the Arm Compiler</description>
-      <require condition="CM1_ARMCC"/>
-      <require Dendian="Big-endian"/>
+    <condition id="Startup ARMCC6 Unsecure">
```

Comment:
Change 'unsecure' to 'Non-secure' because of consistency.

Reply:
I went for unsecure in this place to avoid unambiguity with "non-secure". This condition shall aggregate TrustZone Non-Secure and Non-TrustZone by inverting the TrustZone Secure condition.

Reply:
I would also stick with "non-secure" t be consistent with the condition "TZ Non-secure" which accepts "Non-secure" and "TZ-disabled".
Condition "Startup ARMCC6 Non-secure" could probably use accept "TZ Non-secure" (rather then deny with reverse logic).

Reply:
This condition seems to be kind of a challenge. It must be mutually exclusive with the condition "Startup ARMCC6 Secure". In fact, it needs to be `true` whenever the condition "TZ Secure" is `false`. This is the case if either `Dtz="NO TZ"` or if `Dsecure="Non-secure"` or `Dsecure="TZ-disabled"`. That one could be achived with three `accept` clauses. Is this still "TZ Non-secure"?

Reply:
I see. Then the 'unsecure' makes sense.

Reply:
Shall we stick with `deny` clause to achieve this mutually exclusivity? Or shall we rather switch to three `accept` clauses?